src/SWP/Bundle/CoreBundle/Model/FacebookInstantArticlesArticleInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Component\Common\Model\TimestampableInterface;
+
+interface FacebookInstantArticlesArticleInterface extends TimestampableInterface
```

Comment:
here you should extend from `FacebookInstantArticlesArticleInterface` which should be placed in FacebookInstantArticles Component or bundle and place here only `public function getArticle(): ArticleInterface;` + TenantAwareInterface